File: SignalRDashboard.Web/Scripts/app/models/core/ConnectedUsers.js
(function(core) {
    core.ConnectedUsers = ConnectedUsers;

    function ConnectedUsers() {
        this.numberOfConnectedUsers = 0;
        
        this.updateFromData = function(data) {            
            signalrdashboard.mapping.map(data, this);            
        };
    }
    
})(window.signalrdashboard.core || (window.signalrdashboard.core = {}));

File: SignalRDashboard.Data.Core/Hubs/Models/ConnectedUsers.cs
namespace SignalRDashboard.Data.Core.Hubs.Models
{
    public class ConnectedUsers : DashboardHubModel
    {
        private int _connectedUsers;
        public int NumberOfConnectedUsers
        {
            get { return _connectedUsers; }
            set { SetProperty(ref _connectedUsers, value); }
        }
    }
}
